packages/cli/src/commands/start/startFrontend.ts

```diff
@@ -62,14 +62,30 @@ export async function startFrontend(options: StartAppOptions) {
   }
 
   const { name } = await fs.readJson(paths.resolveTarget('package.json'));
+  const config = await loadCliConfig({
+    args: options.configPaths,
+    fromPackage: name,
+    withFilteredKeys: true,
+  });
+
+  const appBaseUrl = config.frontendConfig.getString('app.baseUrl');
+  const backendBaseUrl = config.frontendConfig.getString('backend.baseUrl');
+  if (appBaseUrl === backendBaseUrl) {
+    console.log(
+      chalk.yellow(
```

Comment:
Separate note, I think the `⚠️` could use an extra space after it since it occupies a bit of extra space.

Reply:
I copied the spacing of the warnings above in the file. Want me to change their spacing too?

Reply:
I copied the spacing of the warnings above in the file. Want me to change their spacing too?

Reply:
Yep, I think that makes sense 👍